readthedocs/api/v2/views/footer_views.py

```diff
@@ -30,7 +30,10 @@ def get_version_compare_data(project, base_version=None):
     :param base_version: We assert whether or not the base_version is also the
                          highest version in the resulting "is_highest" value.
     """
-    if not project.show_version_warning:
+    if (
+        not project.show_version_warning or
+        (base_version and base_version.is_external)
+    ):
         return {'is_highest': False}
```

Comment:
Is there a reason we're setting this to `False` here instead of `True`? Seems if we're opting out of showing it, setting it to `True` makes more sense, even though I know we likely aren't testing for it.

Reply:
Just to avoid running the queries below, that value is only used for the warning message.

Reply:
Gotcha -- just seems like `True` is a better default for other edge cases, but I don't care too much.